src/Slot.ts

```diff
@@ -165,23 +165,35 @@ export function connectSlot<T = void, T2 = void>(
 
     // Signal to all transports that we will accept handlers for this slotName
     transports.forEach((transport, transportKey) => {
-
         const remoteHandlerRegistered = (
             param = DEFAULT_PARAM,
             handler: Handler<any, any>
         ) => {
+            // If the far end had blacklisted an event but is now trying to
```

Comment:
Can you be more specific about "the far end", are you just referring to the other side of the channel?